docs/README.md

```diff
@@ -450,15 +448,15 @@ const {url, state} = cozy.auth.getAuthCodeURL(client, ['files/images:read'])
 
 // save state and redirect to url
 localStorage.setItem("oauthstate", state)
-document.location.replace(url)
+window.location.replace(url)
```

Comment:
I think there are over places where `document.location` is used and should be replaced with `window.location`